bin/nyc.js

```diff
@@ -46,6 +46,11 @@ var yargs = require('yargs/yargs')(process.argv.slice(2))
     type: 'boolean',
     describe: 'whether or not to instrument all files of the project (not just the ones touched by your test suite)'
   })
+  .option('include', {
+    alias: 'in',
+    default: [],
```

Comment:
Also from readme:

> Note: include defaults to ['**']

But this default value probably defined somewhere else in the code. Should we remove it there, and specify `['**']` here. Although we need default value when it's loaded from package.json too, so maybe only specify here and not remove in other place?